index.js

```diff
@@ -586,6 +586,13 @@ class WebUntis {
         });
 
         if (typeof response.data.data !== 'object') throw new Error('Server returned invalid data.');
+
+        if (response.data.data.error && response.data.data.error.data && response.data.data.error.data.messageKey)
```

Comment:
I suggest to use optional chaining.

Reply:
I thought we should keep backwards compatibility for node < 14 because optional chaining isn't used for the rest of the code. But if you want I'll add that and add a min version in the package.json.

Reply:
This is true. But I am not sure why we should keep the backwards compatibility with node 14. Node 14 is in maintenance mode.

Reply:
Yeah, I'm totally fine with either. Just tell me what you want I should do.

Reply:
Then go ahead with optional chaining.

Reply:
I also refactored to use a specific property `code` on the error instead of using the error message for the error code and also refactored the optional chaining below in this function.